core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -144,6 +148,16 @@
   }
 
   /* All classes below should start with .conversation-skin */
+  .conversation-skin-back-to-collection-container {
+    margin-top: 25px;
+    margin-bottom: 92px;
```

Comment:
So, everything seems to be look uniform on desktop, mobile, and tablet. The margins are the same, and the footer height is the same, so no additional style rules are needed, I think.

Mobile Margin Same:
![mobile1](https://user-images.githubusercontent.com/6820349/27165411-572e9088-5159-11e7-901e-ef5a3656859a.png)

Mobile Footer Same:
![mobile2](https://user-images.githubusercontent.com/6820349/27165410-572dcd2e-5159-11e7-8547-dbd6bfc34489.png)

Uniform on Tablet:
![tablet](https://user-images.githubusercontent.com/6820349/27165412-57315246-5159-11e7-9427-77d30f78ee9b.png)